Exercises/Chapter-4.txt

```diff
@@ -0,0 +1,44 @@
+1. Assign variables city and state to your current city and state of residence. (If residing outside the U.S., substitute the analogous quantities.)
+city = "Brooklyn"
+state = "New York"
+
+
+2. Using interpolation, print (using puts) a string consisting of the city and state separated by a comma and as space, as in “Los Angeles, CA”.
+puts "#{city}, #{state}"
+
+3. Repeat the previous exercise but with the city and state separated by a tab character.
+puts "#{city} v\ #{state}" (Joe's comment: I don't know what the fuck a tab charater is.)
+
+
+4. What is the result if you replace double quotes with single quotes in the previous exercise?
+"\#{city} \\v \#{state}"
```

Comment:
you changed the backslash instead of the quoes, so you didn't really get the point of #3 and #4.

double quotes: `"`
single quotes: `'`